contrib/ohi-release-notes/action.yml

```diff
@@ -53,13 +56,23 @@ runs:
       uses: newrelic/release-toolkit/validate-markdown@v1
       with:
         markdown: ${{ inputs.git-root }}/CHANGELOG.md
+    # excluded-dependencies is a file contained by this action and is located inside $GITHUB_ACTION_PATH, but this
+    # folder is not mounted in the container that runs the generate-yaml action, so we copied the file to
+    # ${GITHUB_WORKSPACE} which is mounted in the container.
+    - name: Copy dependencies file to temp folder mounted by the generate action
+      shell: bash
+      run: |
+        if ! [ -f "${GITHUB_WORKSPACE}/${{ inputs.excluded-dependencies }}" ]; then
+          cp ${{ github.action_path }}/excluded-dependencies.yml "${GITHUB_WORKSPACE}/${{ inputs.excluded-dependencies }}"
+        fi
```

Comment:
As discussed offline, for now, we'll go with just this PR, but for future contributions, it's better to re-tag first `v1` and then update the ohi contrib action in a follow-up PR. In this case it was tested but pointing to this branch as commented https://github.com/newrelic/release-toolkit/pull/160#issuecomment-1580794953, but the PR will be merged failing since will point to v1.

Reply:
last time I tested it changing the tag pointing to the local branch and then changing it back